src/goto-instrument/goto_instrument_parse_options.h

```diff
@@ -97,7 +98,8 @@ Author: Daniel Kroening, kroening@kroening.com
   OPT_REMOVE_CALLS_NO_BODY \
   OPT_REPLACE_FUNCTION_BODY \
   OPT_GOTO_PROGRAM_STATS \
-  "(show-local-safe-pointers)(show-safe-dereferences)"
+  "(show-local-safe-pointers)(show-safe-dereferences)" \
+  OPT_REPLACE_CALLS
```

Comment:
put \ at the end, and `// empty last line` in the next line to make diffs more readable in future